ocaml/sdk-gen/java/main.ml

```diff
@@ -569,16 +519,15 @@ let gen_class cls folder =
   print_license file ;
   fprintf file
     "package com.xensource.xenapi;\n\n\
+     import com.fasterxml.jackson.annotation.JsonProperty;\n\
```

Comment:
It could be easier to use the `{| .. |}` string that does not require escapes for newlines

Reply:
I'd go for a templating engine here (mustache is available)

Reply:
yes, it's for another PR, my point was that we can leave it as it is, changing the code to use the new style of quoted strings doesn't seem to be worth the effort as I'd rather do something else in the end